src/lib/features/project/project-store.ts

```diff
@@ -742,15 +742,17 @@ class ProjectStore implements IProjectStore {
                     instances: [row.instance_id],
                     sdks: [getSdk(sdkParts)],
                 };
-                entriesMap.set(row.feature_name, entry);
+                entriesMap.set(row.app_name, entry);
                 orderedEntries.push(entry);
             }
 
             const sdk = entry.sdks.find((sdk) => sdk.name === sdkParts[0]);
             if (!sdk) {
                 entry.sdks.push(getSdk(sdkParts));
             } else {
-                sdk.versions.push(sdkParts[1]);
+                if (!sdk.versions.includes(sdkParts[1])) {
```

Comment:
We never check if sdkParts exists after splitting, can there be a scenario where we don't have it?

Reply:
That is good question, also we might not have the value at all. Probably we need to add more validations in future.